sources/packages/web/src/services/http/StudentAssessmentApi.ts

```diff
@@ -38,4 +39,28 @@ export class StudentAssessmentApi extends HttpBaseClient {
       this.addClientRoot(`assessment/application/${applicationId}/history`),
     );
   }
+
+  /**
+   * Get the NOA values for a student application on a particular assessment.
+   * @param assessmentId assessment id to get the NOA values.
+   * @returns NOA and application data.
+   */
+  async getAssessmentNOA(
+    assessmentId: number,
+  ): Promise<AssessmentNOAAPIOutDTO> {
+    return this.getCallTyped<AssessmentNOAAPIOutDTO>(
+      this.addClientRoot(`assessment/${assessmentId}/noa`),
+    );
+  }
+
+  /**
+   * Confirm Assessment of a Student.
+   * @param assessmentId assessment id to be confirmed.
+   */
+  async confirmAssessmentNOA(assessmentId: number): Promise<void> {
+    await this.patchCall(
```

Comment:
just curious,  can we use `this.getCallTyped` here

Reply:
Sorry, I am not following this, this is a _patch_ method on the API, and we do not have _patchCallTyped_ because the _patchCall_ is already typed. What do you mean by `getCallTyped `?